src/utils/OWLConversor/OWLConversor.js

```diff
@@ -0,0 +1,145 @@
+import './xml2json.js';
+import './Utils.js';
+import './Query.js';
+
+(function(){
+	window.MAX_OBJECTS_PER_ROW = 5;
```

Comment:
I think you don't need to use `window` here.
You can declare this in the top of the file, as a constant:

```js
const MAX_OBJECTS_PER_ROW = 5;
```